sympy/physics/control/tests/test_lti.py

```diff
@@ -263,6 +249,15 @@ def test_TransferFunction_functions():
     assert tf7.xreplace({s: k}) == TransferFunction(a0*k**p + a1*p**k, a2*p - k, k)
     assert tf7.subs(s, k) == TransferFunction(a0*s**p + a1*p**s, a2*p - s, s)
 
+    # Conversion to Expr with _to_expr()
+    tf8 = TransferFunction(a0*s**5 + 5*s**2 + 3, s**6 - 3, s)
+    tf9 = TransferFunction((5 + s), (5 + s)*(6 + s), s)
+    tf10 = TransferFunction(0, 1, s)
+    tf11 = TransferFunction(1, 1, s)
+    assert tf8._to_expr() == Mul((a0*s**5 + 5*s**2 + 3), Pow((s**6 - 3), -1, evaluate=False), evaluate=False)
+    assert tf9._to_expr() == Mul((s + 5), Pow((s**2 + 11*s + 30), -1, evaluate=False), evaluate=False)
+    assert tf10._to_expr() == Mul(sympify(0), Pow(simplify(1), -1, evaluate=False), evaluate=False)
+    assert tf11._to_expr() == Mul(sympify(1), Pow(simplify(1), -1, evaluate=False), evaluate=False)
```

Comment:
I missed these on first pass. Scratch my other comment.

Reply:
What is the purpose of simplify and sympify here? You can use `S(0)` to get a SymPy zero if needed.